CMakeLists.txt

```diff
@@ -203,6 +203,15 @@ if (WITH_PIRANHA)
     set(WITH_BOOST yes)
 endif()
 
+# BENCHMARKS
+set(BUILD_BENCHMARKS yes
+    CACHE BOOL "Build SymEngine benchmarks")
+
+if (BUILD_BENCHMARKS)
+    set(WITH_PTHREAD yes)
```

Comment:
For Windows, we don't need this and since Nonius supports MSVC 2013 and above, we can use it from Windows as well